src/holofuel/components/FlashMessage/FlashMessage.js

```diff
@@ -20,7 +21,14 @@ export default function FlashMessage () {
   if (isEmpty(message)) return null
 
   return <div styleName='flash-message'>
-    <div styleName='message'>{message}</div>
+    <div styleName='flash-body'>
+      {link ? <>
+        <div styleName='message'>{message}</div>
+        <br />
+        <Link to={link}>{linkText}</Link>
+      </>
+        : <div styleName='message'>{message}</div> }
+    </div>
```

Comment:
I'd like to handle this differently.

Instead of having a specific `linkText` and `link` that you can pass in, it should just test if `message` is a string. If so, render it in a div, if not, assume it's jsx and render it directly.

Actually, if you always wrap it in the `'message'` div, you don't even need to check.